src/strife/f_wipe.c

```diff
@@ -88,18 +92,27 @@ wipe_doColorXForm
   int	height,
   int	ticks )
 {
-    byte *cur_screen = wipe_scr;
-    byte *end_screen = wipe_scr_end;
+    pixel_t *cur_screen = wipe_scr;
+    pixel_t *end_screen = wipe_scr_end;
     int   pix = width*height;
     int   i;
     boolean changed = false;
 
+    // [crispy] reduce fail-safe crossfade counter tics
+    fade_safe_tics--;
+
     for(i = pix; i > 0; i--)
     {
-        if(*cur_screen != *end_screen)
+        if(*cur_screen != *end_screen && fade_safe_tics)
```

Comment:
`CROSSFADETICS` will need to be `14`, or else the code will only run 12 times.

Reply:
Done in https://github.com/fabiangreffrath/crispy-doom/pull/1183/commits/93f36f377cd365963256412b44f22eb2708951dc as well. Thanks for pointing out, my oversight.